src/components/radio/radio_group.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
This should be inlined in the component definition, using `template:` instead of `templateUrl:`.